src/develop/masks/brush.c

```diff
@@ -2156,8 +2162,9 @@ static void dt_brush_events_post_expose(cairo_t *cr, float zoom_scale, dt_masks_
     }
     else
     {
-      float masks_border, masks_hardness, masks_density;
-      float radius, oldradius, opacity, oldopacity, pressure;
+      float masks_border = 0.0, masks_hardness = 0.0, masks_density = 0.0;
+      float opacity = 0.0, oldopacity = 0.0, pressure = 0.0;
+      float oldradius_hardness = 0;                          
```

Comment:
0.0f

Reply:
Ok, I think I got them all, plus some old ones for free :-)